django-testbed/django-root/xenopus_frog/views.py

```diff
@@ -7,14 +7,57 @@
 from xenopus_frog.models import User
 
 
+class BunniesSerializer(ModelSerializer):
+    """
+    Regulates fields exposed by default & as requested.
+
+    Construct URLs like /?include_fields=first_name&include_fields=email to opt-in into fields.
```

Comment:
I think this will be too verbose - URL will be massive in lots of cases

I think `include_fields=first-name,email` will be nicer

@levic thoughts?